server/src/main/java/kr/codesquad/library/domain/openapi/response/OpenApiBookDataResponse.java

```diff
@@ -0,0 +1,14 @@
+package kr.codesquad.library.domain.openapi.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+@Getter
+public class OpenApiBookDataResponse {
```

Comment:
OpenApi 라는 단어가 굳이 들어갈 이유가 있을까요?

```suggestion
public class BookDataResponse {
```

Reply:
왜냐하면 인터파크 openApi를 써서 기존의 bookdata와 달라 표시했습니다